src/js/features/conversations.js

```diff
@@ -34,6 +33,9 @@ function Conversations(timeout) {
     if (window.App && App.__container__.lookup('service:whispers-shim')) {
         var whisperShim = App.__container__.lookup('service:whispers-shim');
         whisperShim.on('whisper', _self.onWhisper);
+        whisperShim.on('thread', function(data) {
+            bttv.ws.joinConversation(data.id);
```

Comment:
> called every time you read

can you comment that in, because that's totally not the connotation one would assume here.